### PR TITLE
.gitignore: Add .vscode

### PR DESCRIPTION
**Pull request by bhavishyagopesh**
Update .gitignore to add the meta folder of Visual Studio Code.

Closes https://github.com/coala/coala/issues/4835

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by bhavishyagopesh** [comment]
This fixes #4835 since #4837 was stale.

File: .gitignore [hunk 1]
```diff
@@ -37,3 +37,4 @@ node_modules
 Goopfile.lock
 venv/
 rultor_secrets.sh
+.vscode
```
Comment (jayvdb) [inline .gitignore]: EOL at EOF.

**Comment by jayvdb** [comment]
unack https://github.com/coala/coala/pull/4938/commits/f352e577c926ed82cc4bc17a7d43ac2f0fda40d9

File: .gitignore (hunk 1 above)
Comment (bhavishyagopesh) [reply .gitignore]: fixed

**Comment by mridubhatnagar** [comment]
LGTM. But just a minor change. Though I did leave it upto you to do it or not.
Line mentioned in commit body can be split up into 2 lines instead of 1
long line. It would improve the readability IMO.

**Comment by Makman2** [comment]
ack e949b89

**Pull request merged**